internal/conf/configuration.go

```diff
@@ -399,6 +399,11 @@ type MailerConfiguration struct {
 	OtpLength int  `json:"otp_length" split_words:"true"`
 
 	ExternalHosts []string `json:"external_hosts" split_words:"true"`
+
+	// EXPERIMENTAL: May be removed in a future release.
+	EmailValidationExtended   bool   `json:"email_validation_extended" split_words:"true" default:"false"`
```

Comment:
i thought we were planning of just checking whether the `EmailValidationServiceURL` is set instead of also having a boolean flag here?

Reply:
from internal discussion: preference for having 2 different flags in case one of them doesn't work well, we can toggle it off